src/global/helpers/scroll.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Handle the page scroll, allowing to disable/enable the window scroll avoiding a potential
+ * content shift caused by the disappearance/appearance of the scrollbar.
+ */
+export class ScrollHandler {
+  private _documentScrollTop: number;
+
+  public disableScroll(): void {
+    if (!this._hasScrollbar()) {
+      document.body.style.overflow = 'hidden';
+      return;
+    }
+
+    // Save a reference to the document's current vertical scroll.
+    this._documentScrollTop = document.documentElement.scrollTop;
+
+    // Set the page as fixed to the top and keep showing an "empty" scrollbar by setting "overflow-y" to "scroll".
+    document.body.style.top = `-${this._documentScrollTop}px`;
```

Comment:
I think we need to save all vars and then later re apply the original ones in case the vars were set to a specific one by customer. What do you think?

Reply:
Of course, I had the same thought and just fixed . Thank you :)